File: md/Learning/Callback/callback_example.c
#include <stdlib.h>
#include <stdio.h>

void executer(int input, int (*func_ptr)(int)){
    (*func_ptr)(input);
}
int add_one(int input){
    printf("%d\n", input + 1);
}
int add_two(int input){
    printf("%d\n", input + 2);
}
int main(){
    executer(2, add_one);
    executer(2, add_two);
    return 0;
}

File: content/Learning/Callback/callback_example.py
def executer(input, func):
    return func(input)


def add_one(input):
    print(input + 1)


def add_two(input):
    print(input + 2)


if __name__ == "__main__":
    executer(1, add_one)
    executer(1,add_two)